ethereum/test/TablelandRigs.ts

```diff
@@ -1831,6 +2067,109 @@ describe("Rigs", function () {
           .withArgs(BigNumber.from(tokenId));
       });
 
+      it("Should allow training after being force parked during a training status", async function () {
+        // First, mint a Rig to `tokenOwner`
+        await rigs.setMintPhase(3);
+        const tokenOwner = accounts[4];
+        const tx = await rigs
+          .connect(tokenOwner)
+          ["mint(uint256)"](1, { value: getCost(1, 0.05) });
+        const receipt = await tx.wait();
+        const [event] = receipt.events ?? [];
+        const tokenId = event.args?.tokenId;
+        // Train the Rig
+        await rigs
+          .connect(tokenOwner)
+          ["trainRig(uint256)"](BigNumber.from(tokenId));
+        // Advance 172800 blocks (30 days)
+        await network.provider.send("hardhat_mine", [
+          ethers.utils.hexValue(172800),
+        ]);
+        // Park the Rig as the contract owner
+        await rigs.parkRigAsOwner([BigNumber.from(tokenId)]);
+        // Check pilot is back to untrained
+        const pilotInfo = await rigs["pilotInfo(uint256)"](
+          BigNumber.from(tokenId)
+        );
+        // Check that the pilot is now `0` since training was incomplete
```

Comment:
nit on the comment: Check that the pilot is now `0` since it was force parked while training

Reply:
oops, didn't see this, fixed!